.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,95 @@
+---
+name: "🐛 Bug Report"
+description: Report a bug
+title: "(short issue description)"
+labels: [bug, needs-triage]
+assignees: []
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: What is the problem? A clear and concise description of the bug.
+    validations:
+      required: true
+  - type: textarea
+    id: modelname
+    attributes:
+      label: Model Name
+      description: Provide Model Name
+    validations:
+      required: true
+  - type: textarea
+    id: workloadtype
+    attributes:
+      label: Describe the workload type 
+      description: Please mention workload type (e.g.Inference/Training workload or anything else.)
+    validations:
+      required: true
+  - type: textarea
+    id: instancetype
+    attributes:
+      label: Instance Type
+      description: |
+        Please provide instance type - Inf2.xlarge/trn1.32xlarge/trn2.48xlarge etc?
+    validations:
+      required: true
+  - type: textarea
+    id: release
+    attributes:
+      label: Release version 
+      description: |
+        Provide Neuron SDK release version - ( e.g. 2.25.0) and all relevant component versions. 
+        apt list --installed | grep -i -e neuron,   pip list | grep -i -e neuron -e torch -e transformers -e jax      
```

Comment:
How does this appear when rendered? I'm hoping we can format these commands as a code block with each command on a separate line.